.gitignore

```diff
@@ -10,12 +10,14 @@ __pycache__/
 CMakeLists.txt
 cmake-build-debug
 venv/
+.venv/
```

Comment:
do we need two different venv ignores?

Reply:
You can name your venv folder however you want, both are typically used so it doesn't hurt

Reply:
I prefer using `.venv`  (with a dot) because that's a hidden folder on Linux. I'm not sure if explorer.exe would honor this when examining the WSL filesystem.
I didn't remove the other one in case somebody was already using it